### DISCUSSION TITLE
[Feature] <Support for MTS video files>

### DISCUSSION THREAD

**Discussion by mkjustuk** [immich-app/immich#2734] [Feature Request]
### The feature

I have a Sony camera and it uses their MTS video format.  It would be great to have support for those, so I can upload both the photos and videos it creates using drag and drop via the web app - thanks!

### Platform

- [X] Server
- [ ] Web
- [ ] Mobile

**Comment by wackyesolution** [comment]
Very usefull for old clips

**Comment by uhthomas** [comment]
Duplicate of #2350.

**Reply by mkjustuk** [reply]
Thanks, sorry my search hadn't shown that one.
